packages/puppeteer-core/src/api/Page.ts

```diff
@@ -295,7 +305,7 @@ export interface ScreenshotOptions {
    * relative to current working directory. If no path is provided, the image
    * won't be saved to the disk.
    */
-  path?: string;
+  path?: `${string}.${ImageFormat}`;
```

Comment:
This makes it unnecessarily annoying to use a computed value for `path`. For example, ``await page.screenshot({ path: path.join(dir, `${name}.png`) })`` leads to error TS2769: ``Type 'string' is not assignable to type '`${string}.webp` | `${string}.jpeg` | `${string}.png`'.``